core/src/banking_stage/packet_filter.rs

```diff
@@ -20,10 +22,14 @@ impl ImmutableDeserializedPacket {
     /// which are statically known to exceed the compute budget, and will
     /// result in no useful state-change.
     pub fn check_insufficent_compute_unit_limit(&self) -> Result<(), PacketFilterFailure> {
+        // To calculate the static_builtin_cost_sum conservatively, an all-enabled dummy feature_set
+        // is used. It lowers required minimal compute_unit_limit, aligns with future versions.
+        let feature_set = FeatureSet::all_enabled();
```

Comment:
if this is backported, we should just set up a lazy static here I think. As is, it's re-creating the feature-set (HashSet + HashMap) for every packet; with a new random seed.